lib/cli-engine.js

```diff
@@ -374,7 +374,7 @@ CLIEngine.prototype = {
         if (typeof format === "string") {
 
             // if there's a slash, then it's a file
-            if (format.indexOf("/") > -1) {
+            if (format.indexOf("/") > -1 || format.indexOf("\\") > -1) {
```

Comment:
Where would you do this?
A possible place would be at https://github.com/eslint/eslint/blob/master/lib/cli-engine.js#L371, something like `format = typeof format === "string" && format.replace(/\\/g, "/") || "stylish";`